addons/l10n_es_edi_facturae/models/account_move_send.py

```diff
@@ -57,6 +57,7 @@ def _hook_invoice_document_before_pdf_report_render(self, invoice, invoice_data)
         super()._hook_invoice_document_before_pdf_report_render(invoice, invoice_data)
 
         if self.l10n_es_edi_facturae_checkbox_xml and invoice._l10n_es_edi_facturae_get_default_enable():
+            xml_content = b''
```

Comment:
@paso-odoo I don't think we should return anything anyways when we had an error.  Please compare with the code in https://github.com/odoo/odoo/blob/ed9a1d83bd7d496a316e8da6a201ca3a14c99d19/addons/account_edi_ubl_cii/models/account_move_send.py#L100

cc @JulienAlardot

Reply:
Indeed, otherwise you would still be setting the field `l10n_es_edi_facturae_xml_file` with an empty field, which seems wrong.